common/SC_PaUtils.cpp

```diff
@@ -174,20 +174,23 @@ PaStreamParameters MakePaStreamParameters(int device, int channelCount, double s
     streamParams.hostApiSpecificStreamInfo = &macInfo;
 #elif defined(_WIN32)
     // WASAPI options
-    static PaWasapiStreamInfo wasapiInfo = []() {
-        PaWasapiStreamInfo info;
-        memset(&info, 0, sizeof(info));
-        info.size = sizeof(PaWasapiStreamInfo);
-        info.hostApiType = paWASAPI;
-        info.version = 1;
-        info.flags = paWinWasapiAutoConvert; // automatic sample rate conversion
-        return info;
-    }();
-
     const PaDeviceInfo* info = Pa_GetDeviceInfo(device);
-    const PaHostApiInfo* api = Pa_GetHostApiInfo(info->hostApi);
-    if (api->type == paWASAPI)
+    const PaHostApiInfo* api = info ? Pa_GetHostApiInfo(info->hostApi) : nullptr;
+
+    if (api && api->type == paWASAPI) {
```

Comment:
Small nitpick, but this could really be simplified to:
```
const PaDeviceInfo* info = Pa_GetDeviceInfo(device);
const PaHostApiInfo* api = Pa_GetHostApiInfo(info->hostApi);
if (api->type == paWASAPI) {
```

We got the device index and API index from portaudio, so these are guaranteed to be valid. We never do null-checks on those. If you feel like it, you can change this. Otherwise I will just merge it as is.

Reply:
Sounds good, changing

Reply:
Cool! Will merge now.